src/test/groovy/graphql/GraphQLTest.groovy

```diff
@@ -1363,4 +1363,28 @@ many lines''']
         then:
         ! er.errors.isEmpty()
     }
+
+    def "deprecated execute method works"() {
+        given:
+        def query = """
+        query FetchSomeIDQuery(\$someId: String!) {
+            human(id: \$someId) {
+                name
+            }
+        }
+        """
+        def params = [
+                someId: '1000'
+        ]
+        def expected = [
+                human: [
+                        name: 'Luke Skywalker'
+                ]
+        ]
+        when:
+        def result = GraphQL.newGraphQL(StarWarsSchema.starWarsSchema).build().execute(query, null, params).data // Retain deprecated method for test coverage
```

Comment:
https://github.com/graphql-java/graphql-java/pull/2932 will remove `execute`, I'll remove `execute` tests after that PR is merged

Reply:
merged